apis/v1beta1/config.go

```diff
@@ -75,6 +87,37 @@ func (c *AnyConfig) MarshalJSON() ([]byte, error) {
 	return json.Marshal(c.Object)
 }
 
+// Pipeline is a struct of component type to a list of component IDs.
+type Pipeline struct {
+	Receivers  []string `json:"receivers" yaml:"receivers"`
+	Processors []string `json:"processors" yaml:"processors"`
+	Exporters  []string `json:"exporters" yaml:"exporters"`
+}
```

Comment:
I think the fields are marshalled in the order they're declared in the struct. So if you change the order to alphabetical, we can avoid a bunch of test changes due to ordering.

Reply:
I had put them in this order because of how people conceptually think of a pipeline, but i guess this isn't really meant to be user read, so it doesn't really matter. Will change it back!